packages/framework/client-logger/fluid-telemetry/src/container/telemetryManager.ts

```diff
@@ -4,14 +4,15 @@
  */
 
 import { type ICriticalContainerError } from "@fluidframework/container-definitions";
-import { IContainerTelemetry, type ContainerHeartbeatTelemetry } from "./containerTelemetry.js";
+import { type IContainerTelemetry } from "./containerTelemetry.js";
 import { ContainerEventTelemetryProducer } from "./telemetryProducer.js";
-import { ITelemetryConsumer } from "../common/index.js";
+import { type ITelemetryConsumer } from "../common/index.js";
 import {
-	IFluidContainerSystemEventName,
+	type IFluidContainerSystemEventName,
```

Comment:
Nit (not in this file but this was an available place to point it out 😄). `IFluidContainerSystemEventNames` is an object, not an interface, so let's remove the leading "i".

Reply:
I did this because the events themselves are called `IFluidContainerEvents` but I get how it doesn't make sense. You think I should still update the naming?